pkg/reconciler/serverlessservice/resources/services_test.go

```diff
@@ -388,7 +388,7 @@ func TestMakePrivateService(t *testing.T) {
 	}, {
 		name: "HTTP2 and long",
 		sks: sks(func(s *v1alpha1.ServerlessService) {
-			s.Name = "dream-tonight-cherub-rock-mayonaise-hummer-disarm-rocket-soma-quiet"
+			s.Name = "dream-tonight-cherub-rock-mayonnaise-hummer-disarm-rocket-soma-quiet"
```

Comment:
as with `windermere`, if this is something unrelated to the standard `mayonnaise`, a reference explaining it would be helpful.

Reply:
those are song names, proper one has `nn`, you can fix this.